### PR TITLE
Changed java.util.logging to SLF4J

### PR DESCRIPTION
**Pull request by ghost**
+ Added test-output from testng to .gitignore
+ Replaced java.util.logging with SLF4J
  see: ttps://www.slf4j.org/apidocs/org/slf4j/bridge/SLF4JBridgeHandler.html
+ Used logback as SLF4J implementation

### CODE REVIEW

File: wrapper/src/main/java/com/moodysalem/phantomjs/wrapper/PhantomJS.java [hunk 1]
```diff
@@ -233,28 +256,44 @@ public static PhantomJSExecutionResponse exec(InputStream script, PhantomJSOptio
...
+    /**
+     * Info level logger.
+     */
+    private static class InfoLoggerOutputStream extends LoggerOutputStream {
+
         @Override
-        protected void processLine(String s, int i) {
-            logger.log(level, String.format("PhantomJS script logged: %s", s));
+        protected void processLine(final String s, final int i) {
+            logger.info("PhantomJS script logged: {}", s);
```
Comment (moodysalem) [inline wrapper/src/main/java/com/moodysalem/phantomjs/wrapper/PhantomJS.java]: missing a %s here

**Comment by moodysalem** [comment]
hey @autumn-moon would you mind describing the improvement of using slf4j over JUL? I know it's probably been described elsewhere but it's nice to keep out dependencies that you don't necessarily need-I imagine if you wanted to use slf4j you could use the bridge handler, not sure how it's used in this PR

File: wrapper/src/main/java/com/moodysalem/phantomjs/wrapper/PhantomJS.java (hunk 1 above)
Comment (autumn-moon) [inline wrapper/src/main/java/com/moodysalem/phantomjs/wrapper/PhantomJS.java]: The curly braces are the wildcard for what is %s in String.format()
Did you mean that or did I miss some variable?

**Comment by ghost** [comment]
Hi @moodysalem,

first off thank you for taking your time reviewing my PR.
In my opinion SLF4J became a de facto standard for logging. It's a neat facade for any logging framework which supports SLF4J. So if you are not a fan of logback you might just change the dependency to log4j2 for example. Instead of using multiple logging configs for various frameworks you can just have your SLF4J config. It's a bliss.
In my opinion did SLF4J syntax improves the readability drastically. For simple log statements using a variable you don't have to make any checks `isDebugEnabled()` or the like. However if you use something like `logger.debug("text {}", var.someMethod());` I strongly recommend to still encapsulate constructs like this with a `isLEVELEnabled()`, because the method call will be executed first during runtime and you might have an expensive method call here. Worst case is that this expensive call was executed first before the log level has been checked within the `debug()` or whatever method.
I also heard that the SLF4J Frameworks like logback and log4j2 provide better performance and that they are more garbage collection friendly. You might find other arguments elsewhere ;)

**Comment by moodysalem** [comment]
@autumn-moon Thank you, I will take a look later today

**Comment by moodysalem** [comment]
3.1.1 should have this

**Pull request merged**